enclave-runtime/src/lib.rs

```diff
@@ -578,6 +582,79 @@ where
 	Ok(())
 }
 
+/// For now get the DOT/currency exchange rate from coingecko API.
+#[no_mangle]
+pub unsafe extern "C" fn update_market_data_xt(
+	genesis_hash: *const u8,
+	genesis_hash_size: u32,
+	currency: *const u8,
+	currency_size: u32,
+	unchecked_extrinsic: *mut u8,
+	unchecked_extrinsic_size: u32,
+) -> sgx_status_t {
+	let genesis_hash_slice = slice::from_raw_parts(genesis_hash, genesis_hash_size as usize);
+	let genesis_hash = hash_from_slice(genesis_hash_slice);
+
+	let mut curr_slice = slice::from_raw_parts(currency, currency_size as usize);
+	let curr: String = Decode::decode(&mut curr_slice).unwrap();
```

Comment:
Just as a remark for code readability: We shouldn't use these abbreviated variable names, it impairs the readability of the code. Name it `currency_slice` and `currency`. And the parameter `currency_ptr` (where `ptr` is a mostly accepted abbreviation for pointer) and `currency_size`.

Reply:
I applied here the logic used in multiple other places. As this code will not be merged into the master branch, I proceed to the changes only in this method. I think we should create a new issue to update the variable name through the code according your remark.

Reply:
Yes I think we'll have to live with it for now. From my experience we have to start at some point and do this transition gradually inside other PRs (and accept inconsistencies for a time). An issue with the description 'Change/improve variable names' will never make it into a sprint.

Reply:
No, but to avoid that the old logic is applied in the future, we should rework it as soon as possible, at least when someone is working on similar code. As this code will not be merged into the master, it would not inspired the next developer working on "xt"